webtau-junit4-examples/src/test/java/com/example/tests/junit4/CustomerCrudSingleJavaTest.java

```diff
@@ -14,6 +15,12 @@
     private Map<String, Object> customerPayload = createCustomerPayload();
     private Map<String, Object> changedCustomerPayload = createChangedCustomerPayload();
 
+    @BeforeClass
+    public static void setUpClass() {
+        String port = System.getProperty("springboot.http.port", "8080");
```

Comment:
do we still need 8080 here?

Reply:
I added the default value to make debugging in an IDE easier because the port randomisation happens in Maven.  If you want to run the tests in an IDE, you'd just start the springboot app which will default to 8080 so it's useful to then just run these tests without having to set a system prop.

Reply:
ah, I didn't realize it is a second parameter. somehow just saw it as the only value